base_rest_pydantic/restapi.py

```diff
@@ -55,15 +45,20 @@ def from_params(self, service, params):
 
     def to_response(self, service, result):
         # do we really need to validate the instance????
-        json_dict = result.dict()
-        if PYDANTIC_V2:
-            orm_mode = result.model_config.get("from_attributes", None)
-        else:
-            orm_mode = result.__config__.orm_mode
-        to_validate = json_dict if orm_mode else result.dict(by_alias=True)
-        *_, validation_error = validate_model(self._model_cls, to_validate)
-        if validation_error:
-            raise SystemError(_("Invalid Response %s") % validation_error)
+        json_dict = result.model_dump()
```

Comment:
Not sure the name `json_dict` is still relevant as we need to convert it into json at line 54